README.md

```diff
@@ -100,7 +100,7 @@ So this is the thinner wrapper we can came with to make Caffeine easy and idioma
 Download from [Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.blemale/scaffeine_2.11) or depend via SBT:
 
 ```sbt
-"com.github.blemale" %% "scaffeine" % "2.1.0" % "compile"
+"com.github.blemale" %% "scaffeine" % "2.2.0" % "compile"
```

Comment:
Don't update the README in this PR, I will do it when I will release the new version.